dandi/cli/command.py

```diff
@@ -0,0 +1,223 @@
+                    #'annex_local_size': 'annex(present)',
+                    #'annex_worktree_size': 'annex(worktree)',
+                }.get(x, x).upper()
+            )),
+            # ('default_', dict(align="center")),
+            ('default_', dict(missing="")),
+            ('path', dict(
+                bold=True,
+                align="left",
+                underline=True,
+                width=dict(
+                    truncate='left',
+                    max=max_filename_len + 1
+                ),
+                aggregate=lambda _: "Summary:"
+                # TODO: seems to be wrong
+                # width='auto'
+                # summary=lambda x: "TOTAL: %d" % len(x)
+            )),
+            # ('type', dict(
+            #     transform=lambda s: "%s" % s,
+            #     aggregate=counts,
+            #     missing='-',
+            #     # summary=summary_counts
+            # )),
+            # ('describe', dict(
+            #     transform=empty_for_none)),
+            # ('clean', dict(
+            #     color='green',
+            #     transform=fancy_bool,
+            #     aggregate=mapped_counts({False: fancy_bool(False),
+            #                              True: fancy_bool(True)}),
+            #     delayed="group-git"
+            # )),
+            ('size', pyouts.size_style),
+            ('session_start_time', dict(
+                transform=pyouts.datefmt,
+                aggregate=pyouts.summary_dates,
+                # summary=summary_dates
+            ))
+        ]
+    )
+    if not sys.stdout.isatty():
+        # TODO: ATM width in the final mode is hardcoded
+        #  https://github.com/pyout/pyout/issues/70
+        # and depending on how it would be resolved, there might be a
+        # need to specify it here as "max" or smth like that.
+        # For now hardcoding to hopefully wide enough 200 if stdout is not
+        # a tty
+        PYOUT_STYLE['width_'] = 200
+
+    out = pyout.Tabular(
+        columns=[
+            'path',
+            'size',
+            #'experiment_description',
+            #'lab',
+            'experimenter',
+            'session_id',
+            'session_start_time',
+            #'identifier',  # note: required arg2 of NWBFile
+            #'institution',
+            'keywords',
+            #'related_publications',
+            #'session_description',  # note: required arg1 of NWBFile
+        ],
+        style=PYOUT_STYLE
+        # , stream=...
+    )
+    with out:
+        for path in files:
+            rec = {'path': path}
+            try:
+                rec['size'] = os.stat(path).st_size
+                rec.update(get_metadata_pyout(path))
+            except FileNotFoundError as exc:
+                #lgr.error("File is not available: %s", exc)
+                pass
+            out(rec)
+        out.wait()
+
+#
+# Validate
+#
+
+@main.command()
+@click.argument('paths', nargs=-1, type=click.Path())
```

Comment:
Should this be `click.Path(exists=True, dir_okay=False)` ?

Reply:
I think so (at least for now, until I add directories treatment) - I will change it now. 
As you can feel I am not a `click` expert (yet). Thanks!